test/runtests.jl

```diff
@@ -1,10 +1,52 @@
 module ImageFeatureTests
 
-using FactCheck, ImageFeatures, Base.Test, Images
+using FactCheck, ImageFeatures, Base.Test, TestImages, Distributions, ColorTypes, Images
 
-include("corner.jl")
+
+function check_samples(sample_one, sample_two, size::Int, window::Int)
+    check_bool = true
+    check_bool = check_bool && length(sample_one) == size
+    check_bool = check_bool && length(sample_two) == size
+    for s1 in sample_one, s2 in sample_two
+        check_bool = check_bool && (s1[1] >= ceil(-window / 2) && s1[1] <= floor((window - 1) / 2)) && (s1[2] >= ceil(-window / 2) && s1[2] <= floor((window - 1) / 2))
+        check_bool = check_bool && (s2[1] >= ceil(-window / 2) && s2[1] <= floor((window - 1) / 2)) && (s2[2] >= ceil(-window / 2) && s2[2] <= floor((window - 1) / 2))
+    end
+    return check_bool 
+end
+
+function _warp(img, transx, transy)
+    res = zeros(eltype(img), size(img))
```

Comment:
Also, https://github.com/timholy/AffineTransforms.jl is better as a base for `CameraGeometry.jl`.